README.md

```diff
@@ -1 +1,14 @@
-# nhsuk-tools-chart-components
\ No newline at end of file
+# NHS.UK Tools Chart components
+
+This repository contains the code for NHS.UK React components - a port of the [NHS.UK Tools Chart components](https://github.com/NHSDigital/nhsuk-tools-chart-components).
+
+# Introduction
+
+This package contains React components for use in various NHS UK Tools projects.
+
+## Installation
+
+Ensure you have Node.js 18.x.x. installed. Usage of NVM is recommended, to allow for easily switching between versions on different projects.
```

Comment:
shall we move to 20? 

Could we add a few more headings into here which run through common package commands

Reply:
updated